core/integer/bit_or_spec.rb

```diff
@@ -30,8 +30,8 @@
     end
 
     it "coerces the rhs and calls #coerce" do
-      obj = mock("fixnum bit and")
-      obj.should_receive(:coerce).with(6).and_return([3, 6])
+      obj = mock("fixnum bit or")
```

Comment:
I think this is _supposed_ to say "fixnum bit and" because of the operation below?

```
6 & obj
```

Reply:
Given the filename `bit_or_spec`, I think the operation on line 35 is wrong as well.

Reply:
I pushed a new version, where line 35 is fixed.